nova/compute.sls

```diff
@@ -114,11 +114,16 @@ user_nova_bash:
     - pkg: nova_compute_packages
 
 nova_compute_services:
+{%- if not grains.get('noservices', False) %}
   service.running:
   - enable: true
   - names: {{ compute.services }}
   - watch:
     - file: /etc/nova/nova.conf
+{%- else %}
```

Comment:
IMHO disabling services is not the purpose of special noservices grain. It's purpose is to avoid touching services in any way.